### DISCUSSION TITLE
New Version v1.3.2. - BLTouch - 4.2.2. - Leveling help!!

### DISCUSSION THREAD

**Discussion by GeoStork** [Jyers/Marlin#677] [Q&A] [answered]
Ok, downloaded and installed the latest firmware version.  Are all versions UBL now or is this menu labeled incorrectly?  I downloaded the same 5x5 for BL Touch (standard bed leveling)... but now I'm confused by this detailed menu.   I'm assuming "Create Mesh" just probes the bed like before but what's "Leveling Active" and "Manual Tuning" and "Leveling Tuning"??   I guess I'll poke around but may just roll back two versions.  

![IMG_8949](https://user-images.githubusercontent.com/78649783/116828865-840b8880-ab6f-11eb-86d4-207b1a3c43cc.jpg)

**Comment by Jyers** [comment, marked as answer]
The fact that the menu is labeled UBL is a bug I fixed soon after release, so if you redownload the bin it should be fixed. Leveling active is basically M420 S1/S0 no real need to worry about it unless you need to print something without that in the start gcode. Manual tuning allows you to modify the values of individual probe points manually if you need to and Leveling settings just has some extra settings like fade height

**Reply by GeoStork** [reply]
Thanks. I will do that.   Good luck on exams!  When are you getting a Patreon account?  You are sponsor worthy!  Take care.

**Reply by Jyers** [reply]
Thanks! And funny you say that... I was just setting one up today. I'll add it to the sponsor button 😄

**Reply by duraluminium** [reply]
Could you please clarify the information in the mesh viewer, am I correct in thinking that green squares are high points and red squares low points? Are the offset values like 0.03 in millimeters? What is taken as the 0.00 point, the point closest to the 'middle' offset? I tried relevelling manually in order to have most of the squares black with 0.01 in them, but that resulted in a fairly bad setting for some reason. Thanks!

**Reply by Jyers** [reply]
Yes, as the key at the bottom notes, green is positive or high, red is negative or low. Everything is in mm, I believe the 0 point is either the average offset or based on the homing point.
